native/packages/react-native-bpk-component-progress/src/styles.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { StyleSheet, Platform } from 'react-native';
+import {
+  colorGray100,
+  colorGreen500,
+  colorGreen50,
+  borderRadiusPill,
+  spacingXxl,
+} from 'bpk-tokens/tokens/base.react.native';
+
+export default {
+  Default: StyleSheet.create({
```

Comment:
I'd try and keep these styles in the main component file - they seem small enough to do so!

Reply:
will do that